akka-sample-sharding-scala/README.md

```diff
@@ -15,6 +15,8 @@ types on each clustered node.
 
 ### Aggregator - sharded data by type
  
+FIXME this is not how it was working though, it would always create one Aggregate per weather station id
```

Comment:
Since how it actually works was to always report data to one wsid and not aggregate in any way I renamed aggregate to WeatherStation

Reply:
Updated to describe how it works after the changes